packages/react/src/analytics/integrations/GA4/eventMapping.ts

```diff
@@ -370,6 +370,7 @@ const getCheckoutPaymentStepParametersFromEvent = (
   return {
     ...getCheckoutParametersFromEvent(eventProperties),
     payment_type: eventProperties.paymentType,
+    transaction_id: eventProperties.orderId,
```

Comment:
I saw that when `transaction_id` field are mapped, always has `getCheckoutParametersFromEvent` too.

What do you think about add `transaction_id` field to `getCheckoutParametersFromEvent` helper. From 12 matches, we can cut half of them by moving to this helper.

Reply:
Sounds good to me! Just updated the code with those changes 👍🏼